src/utils/common.ts

```diff
@@ -309,7 +308,7 @@ export function removeDuplicatedTransactions(transactions: ITransaction[]) {
 }
 
 export function secondsToRelativeTime(seconds: number, shortForm?: boolean) {
-  const { t } = useI18n();
+  const t = tg as any;
```

Comment:
```suggestion
  const t = tg as ComposerTranslation;
```
What do you think about this?